modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -16,11 +16,6 @@ A migration to the OVN-Kubernetes cluster network provider is supported on insta
 * {rh-openstack-first}
 * VMware vSphere
 
-[NOTE]
```

Comment:
It seems only mentioned IPI support here "A migration to the OVN-Kubernetes cluster network provider is supported on installer-provisioned clusters on the following platforms", suggest adding UPI support here too.

Reply:
@huiran0826 thanks!